<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDockWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>-2</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Home Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="imghome">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #BDBDBD;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #BDBDBD;
border-radius: 12;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="metroimg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>640</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Egypt Metro</string>
     </property>
    </widget>
    <widget class="QLabel" name="adminimg">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>620</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Manage Matro</string>
     </property>
    </widget>
    <widget class="QLabel" name="manimg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">text-allignment:left;</string>
     </property>
     <property name="text">
      <string>Tickets</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Subscription</string>
     </property>
    </widget>
    <widget class="QLabel" name="subimg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>49</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>520</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Station</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>691</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 25;</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sign up Requests</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User Accounts</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>691</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 25;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="userhistorybutton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>hallo allllll</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
